Added phone number FAQ to TikTok docs (#899)
rudderlabs/rudderstack-docs: docs/destinations/streaming-destinations/tiktok-ads.mdx

### I see a 40002 response code with ‘No permission to operate pixel code’ error message, what can I do?

> This error code is related to the authentication of your TikTok account. For more information, refer <a href="https://ads.tiktok.com/gateway/docs/index?identify_key=2b9b4278e47b275f36e7c39a4af4ba067d088e031d5f5fe45d381559ac89ba48&language=ENGLISH&doc_id=1701890979375106#item-link-FAQ">TikTok documentation</a>.

Reviewer: ```suggestion
This error code is related to the authentication of your TikTok account. For more information, refer to the <a href="https://ads.tiktok.com/gateway/docs/index?identify_key=2b9b4278e47b275f36e7c39a4af4ba067d088e031d5f5fe45d381559ac89ba48&language=ENGLISH&doc_id=1701890979375106#item-link-FAQ">TikTok documentation</a>.
```

### How does RudderStack validate the phone number?

> RudderStack accepts the phone number with correct country code where it's not `+86`. Also, RudderStack doesn't support any non-numeric values in the phone number. You can read more about it in this [TikTok Documentation](https://ads.tiktok.com/marketing_api/docs?id=1701890979375106).

Reviewer: ```suggestion
RudderStack accepts the phone number with correct country code where it's not `+86`. Also, it does not support any non-numeric values in the phone number. For more information, refer to this [TikTok Documentation](https://ads.tiktok.com/marketing_api/docs?id=1701890979375106).
```

> RudderStack accepts the phone number with correct country code where it's not `+86`. Also, it does not support any non-numeric values in the phone number. For more information, refer to this [TikTok Documentation](https://ads.tiktok.com/marketing_api/docs?id=1701890979375106).

Reviewer: ```suggestion
RudderStack accepts a phone number with the correct country code that's not `+86`. Also, it does not support any non-numeric values in the phone number. For more information, refer to this [TikTok Documentation](https://ads.tiktok.com/marketing_api/docs?id=1701890979375106).
```
